spec/codecs/cef_spec.rb

```diff
@@ -452,7 +452,7 @@ def validate(e)
       end
     end
 
-    let (:trim_additional_fields_with_dot_notations) {'CEF:0|security|threatmanager|1.0|100|trojan successfully stopped|10|src=10.0.0.192 dst=12.121.122.82 ad.field[0]=field0 ad.name[1]=new_name'}
+    let (:trim_additional_fields_with_dot_notations) {'CEF:0|security|threatmanager|1.0|100|trojan successfully stopped|10|src=10.0.0.192 ad.Authentification=MICROSOFT_AUTHENTICATION_PACKAGE_V1_0 ad.Error_,Code=3221225578 dst=12.121.122.82 ad.field[0]=field0 ad.name[1]=new_name'}
```

Comment:
I'll update the test specs; thanks for the feedback @breml.

 Also the dots; it only affects the decoding of CEF and the additional fields; prefixed with ad.<fieldNames> are not used as noted in the paper here : https://kc.mcafee.com/resources/sites/MCAFEE/content/live/CORP_KNOWLEDGEBASE/78000/KB78712/en_US/CEF_White_Paper_20100722.pdf (page 13 and 14) ALSO; 

In ArcSight's FlexConnector developer guide, if any of additional field values are needed for usage/correlation, it'll need to be mapped to one of the many documented fields like deviceVendor, etc. In any case the code I've authored only checks for the notation of ad[dot] and could make it as a config for future if users request for it.